tests/cancun/eip4844_blobs/test_blob_txs.py

```diff
@@ -746,7 +746,7 @@ def test_invalid_normal_gas(
 @pytest.mark.parametrize(
     "tx_error", [TransactionException.TYPE_3_TX_MAX_BLOB_GAS_ALLOWANCE_EXCEEDED], ids=[""]
 )
-@pytest.mark.valid_from("Cancun")
+@pytest.mark.valid_until("Cancun")
```

Comment:
I think we should still run the tests in Prague, it's just a matter that they are no longer going to be invalid. But I understand that the naming of the test might be confusing so maybe bring this exact test over to the Prague folder could be an alternative?

Reply:
Agreed! I have this on another branch already. Going to keep this PR framework focused.